src/modules/chat_moderator_cards/index.js

```diff
@@ -0,0 +1,426 @@
+const $ = require('jquery');
+const moment = require('moment');
+const watcher = require('../../watcher');
+const settings = require('../../settings');
+const twitch = require('../../utils/twitch');
+const twitchAPI = require('../../utils/twitch-api');
+const keyCodes = require('../../utils/keycodes');
+const nicknames = require('../chat_nicknames');
+const dragDomElement = require('../../utils/drag-dom-element');
+
+const VIEWER_CARD = '.chat-room__viewer-card';
+const ROW_CONTAINER_SELECTOR = '.chat-room__viewer-card .viewer-card__actions';
+const VIEWER_CARD_CLOSE = '.viewer-card__hide button';
+const CHAT_LINE_SELECTOR = '.chat-line__message';
+const CHAT_LINE_USERNAME_SELECTOR = '.chat-line__username';
+const VIEWER_LIST_USERNAME_SELECTOR = '.chat-viewers-list__button';
+const MENTION_SELECTOR = '[data-a-target=chat-message-mention]';
+const CHAT_INPUT_SELECTOR = '.chat-input textarea';
+
+const BTTV_MOD_CARDS_ID = 'bttv-mod-cards';
+const BTTV_MOD_CARDS_SELECTOR = `#${BTTV_MOD_CARDS_ID}`;
+const BTTV_MOD_SECTION_CLASS = 'bttv-mod-section';
+const BTTV_OWNER_SECTION_CLASS = 'bttv-owner-section';
+const BTTV_HIDE_SECTION_CLASS = 'bttv-hide-section';
+const BTTV_USER_MESSAGES_ID = 'bttv-user-messages';
+
+const BTTV_ACTION_CLASS = 'bttv-mod-action';
+const BTTV_ACTION_SELECTOR = `.${BTTV_ACTION_CLASS}`;
+const BTTV_ACTION_ATTR = 'bttv-action';
+const BTTV_ACTION_VAL_ATTR = 'bttv-val';
+
+const ACTIONS = {
+    BAN: 'BAN',
+    UNBAN: 'UNBAN',
+    MOD: 'MOD',
+    UNMOD: 'UNMOD',
+    TIMEOUT: 'TIMEOUT',
+    PERMIT: 'PERMIT',
+    MESSAGES: 'MESSAGES'
+};
+
+const ACTIONS_MAP = {
+    BAN: '/ban',
+    UNBAN: '/unban',
+    MOD: '/mod',
+    UNMOD: '/unmod',
+    TIMEOUT: '/timeout',
+    PERMIT: '!permit'
+};
+
+const svgCheck = '<figure class="tw-svg"><svg class="tw-svg__asset tw-svg__asset--followcheck tw-svg__asset--inherit" width="16px" height="16px" version="1.1" viewBox="0 0 16 16" x="0px" y="0px"><path clip-rule="evenodd" d="M6.5,12.75L2,8.25l2-2l2.5,2.5l5.5-5.5l2,2L6.5,12.75z" fill-rule="evenodd"></path></svg></figure>';
```

Comment:
business logic of the moderator card itself (all of its rendering/templating) should be in a separate file.

additionally, you should mutate their moderator card html rather than rebuilding it yourself

Reply:
@night what do you mean by mutating? I'm adding my templates to the "viewer_card" component, not rebuilding it. Unless you mean the part with the moderator buttons?